### PR TITLE
Add support for a few more ktx pixel formats

### PR DESCRIPTION
**Pull request by slimbuck**
This PR adds support for a few more ktx pixel formats.

### CODE REVIEW

File: src/resources/parser/texture/ktx.js [hunk 1]
```diff
@@ -73,99 +92,86 @@ class KtxParser {
...
             return null;
         }
 
-        // Byte offset locating the first byte of texture level data
-        let offset = (16 * 4) + header.bytesOfKeyValueData;
+        // offset locating the first byte of texture level data
+        let offset = 16 + header.bytesOfKeyValueData / 4;
 
+        const format = KNOWN_FORMATS[header.glInternalFormat];
+        const isCubemap = (header.numberOfFaces > 1);
```
Comment (mvaligursky) [inline src/resources/parser/texture/ktx.js:152]: no need for brackets here

Comment (slimbuck) [reply src/resources/parser/texture/ktx.js:152]: yep, but I prefer them?

File: src/resources/parser/texture/ktx.js [hunk 2]
```diff
@@ -73,99 +92,86 @@ class KtxParser {
...
             // #endif
             return null;
         }
 
-        // Byte offset locating the first byte of texture level data
-        let offset = (16 * 4) + header.bytesOfKeyValueData;
+        // offset locating the first byte of texture level data
+        let offset = 16 + header.bytesOfKeyValueData / 4;
 
+        const format = KNOWN_FORMATS[header.glInternalFormat];
```
Comment (mvaligursky) [inline src/resources/parser/texture/ktx.js:151]: move this few lines up to avoid doing it twice

**Review by mvaligursky — APPROVED** [review]
approved with two minor comments

File: src/resources/parser/texture/ktx.js (hunk 2 above)
Comment (slimbuck) [reply src/resources/parser/texture/ktx.js]: yes! thanks

**Pull request merged**